litdata/streaming/resolver.py

```diff
@@ -333,7 +333,7 @@ def _execute(
         )
         if not has_printed:
             cloud_url = os.getenv("LIGHTNING_CLOUD_URL", "https://lightning.ai").replace(":443", "")
-            job_url = f"{cloud_url}/{studio.owner}/{studio._teamspace.name}"
+            job_url = f"{cloud_url}/{studio.owner.name}/{studio._teamspace.name}"
```

Comment:
This might not work when the owner isn't an organization. We need to add a check on the owner type. And add a test.